src/components/MenuBar/MenuBar.js

```diff
@@ -0,0 +1,118 @@
+import React, {useState} from 'react'
+import PropTypes from 'prop-types';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import MenuIcon from '@material-ui/icons/Menu';
+import IconButton from '@material-ui/core/IconButton';
+import { Collapse, Typography } from '@material-ui/core';
+import { navigate } from 'gatsby';
+
+import { withStyles } from '@material-ui/core/styles';
+
+import ButtonBar from './ButtonBar';
+
+const styles = theme => {
+	/**
+	 * This is the limit of the screensize where the MenuBar
+	 * should switch between desktop and mobile.
+	 */
+	const menuBarAdaptiveThreshold = theme.breakpoints.values.sm * 1.3;
+	return {
+		logohome: {
+			display: 'flex',
+			alignItems: 'center',
+			justifyContent: 'center',
+			cursor: 'pointer'
+		},
+		toolbar: {
+			backgroundColor: 'white',
+			justifyContent: 'space-between',
+			padding: '0% 6%'
+		},
+		logo: {
+			height: '1.8rem',
+			marginRight: theme.spacing(1)
+		},
+		wordmark: {
+			height: '1.2rem'
+		},
+		menubtn: {
+			// desktop
+			display: 'none',
+			[theme.breakpoints.down(menuBarAdaptiveThreshold)]: {
+				// mobile
+				display: 'inline'
+			}
+		},
+		desktopMenuBar: {
+			// desktop
+			display: 'inline',
+			[theme.breakpoints.down(menuBarAdaptiveThreshold)]: {
+				// mobile
+				display: 'none'
+			}
+		},
+		mobileMenuBar: {
+			// desktop
+			display: 'none',
+			[theme.breakpoints.down(menuBarAdaptiveThreshold)]: {
+				// mobile
+				display: 'inline'
+			},
+			// default styling
+			boxShadow: theme.shadows[4],
+			position: 'fixed',
+			zIndex: '1',
+			overflow: 'hidden',
+			backgroundColor: theme.palette.grey[100],
+			width: '100%'
+		},
+		mobileBtnContainer: {
+			display: 'flex',
+			flexDirection: 'column',
+			padding: '3%'
+		}
+	};
+};
+function MenuBar({ classes }) {
```

Comment:
This file uses React Hooks, and thus should use the newer `useStyles` pattern rather than the `withStyle` pattern. Concretely, this should be a few changes:

1. Change the `const styles = (theme) => { ... };` to `const useStyles = makeStyles((theme) => { ... });`. See https://github.com/uclaacm/hoth.uclaacm.com/blob/a211aba2b32de50ed3bef48353feacee13b95bfa/src/components/HomePage/HothDescription.js#L13-L17

2. Apply the following change:
   ```suggestion
function MenuBar() {
  const classes = useStyles();
```

3. Remove `MenuBar.propTypes`.

4. Change `export default withStyles(styles)(MenuBar);` to just `export default MenuBar;`.